include/igraph_paths.h

```diff
@@ -34,6 +34,8 @@
 
 __BEGIN_DECLS
 
+typedef igraph_error_t igraph_astar_heuristic_t(igraph_real_t *result, igraph_integer_t vertex_id, void *extra);
```

Comment:
Would be nice to use a name for this type that makes it immediately obvious that it's a function.